<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConversionVelocidad</class>
 <widget class="QDialog" name="ConversionVelocidad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>502</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>962</width>
    <height>502</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>962</width>
    <height>502</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CONVERSIONES VELOCIDAD</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>241</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vL_velocimetro"/>
  </widget>
  <widget class="QSlider" name="vS_control_velocidad">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>22</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="cB_conversiones_velocidad">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 1.5px solid  rgb(0, 170, 255);
font-weight: bold;</string>
   </property>
   <item>
    <property name="text">
     <string>km/h a m/s</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>km/h a mi/h</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>m/s a km/h</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>m/s a mi/h</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mi/h a km/h</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>mi/h a m/s</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_convertir_velocidad">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid  rgb(0, 211, 211);
font-weight: bold;</string>
   </property>
   <property name="text">
    <string>Convertir</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>300</y>
     <width>123</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QRadioButton" name="rB_v_km">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Velocidad en km/h</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rB_v_m">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Velocidad en m/s</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rB_v_millas">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Velocidad en millas/h</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lb_animacion_velocidad">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>531</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btn_volver">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>450</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 2px solid  rgb(255, 0, 0);
font-weight: bold;</string>
   </property>
   <property name="text">
    <string> VOLVER</string>
   </property>
   <property name="icon">
    <iconset resource="imagen.qrc">
     <normaloff>:/new/prefix1/recursos/icono_volver.png</normaloff>:/new/prefix1/recursos/icono_volver.png</iconset>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>425</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cambiar rapidez&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>270</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cambiar unidades de medida&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imagen.qrc"/>
 </resources>
 <connections/>
</ui>
